types/i18n-js/i18n-js-tests.ts

```diff
@@ -17,6 +17,8 @@ I18n.t("noun", { defaultValue: "I'm a {{noun}}", noun: "Mac" });
 I18n.t("some.missing.scope", { defaults: [{ scope: "some.existing.scope" }] });
 I18n.t("some.missing.scope", { defaults: [{ message: "Some message" }] });
 
+I18n.getFullScope("translation", { scope: "some.scoped" });
```

Comment:
Unfortunately, it looks undocumented internal method, which should not be appear in d.ts .

Reply:
It's true it is undocumented, however is exposed and unlike `lookup` is not clearly marked as for internal use only in the source code. I use for example in my `missingTranslation` callback to log the actual scope.